mw-webapp/src/model/businessModel/PlanForNextPeriod.ts

```diff
@@ -18,20 +18,20 @@ export class PlanForNextPeriod {
   /**
    * How long was the job done
    */
-  public time: Time;
+  public time: number;
```

Comment:
why?

Reply:
Change properties for make it similar as in firebase collections. Maybe later will add nested Time class